tests/integration/ceph_upgrade_test.go

```diff
@@ -137,6 +137,24 @@ func (s *UpgradeSuite) TestUpgradeToMaster() {
 	objectStoreName := "upgraded-object"
 	runObjectE2ETestLite(s.helper, s.k8sh, s.Suite, s.namespace, objectStoreName, 1, false)
 
+	logger.Infof("Initializing object bucket claim before the upgrade")
+	bucketStorageClassName := "rook-smoke-delete-bucket"
+	bucketPrefix := "generate-me" // use generated bucket name for this test
+	cobErr := s.helper.BucketClient.CreateBucketStorageClass(s.namespace, objectStoreName, bucketStorageClassName, "Delete", region)
+	require.Nil(s.T(), cobErr)
+	cobcErr := s.helper.BucketClient.CreateObc(obcName, bucketStorageClassName, bucketPrefix, maxObject, false)
+	require.Nil(s.T(), cobcErr)
+	defer func() {
+		s.helper.BucketClient.DeleteObc(obcName, bucketStorageClassName, bucketPrefix, maxObject, false)
+		s.helper.BucketClient.DeleteBucketStorageClass(s.namespace, objectStoreName, bucketStorageClassName, "Delete", region)
+	}()
+
+	// this is also waiting for the object store to be created
```

Comment:
Doesn't the `runObjectE2ETestLite()` wait for the object store initialization? Hopefully we can just keep the OBC retry very short instead of 4 minutes.

Reply:
In the last run, it was ~10 seconds of actual waiting. I do want to have a long enough wait time to make sure we don't get random failures, but I can definitely lower this. Hopefully a minute is good.

Reply:
Oh wait, I missed that this runs a lite E2E object test. Yeah, we can just wait a few seconds here.